exchanges/binance/binance_ufutures.go

```diff
@@ -94,6 +94,10 @@ func (b *Binance) UFuturesOrderbook(ctx context.Context, symbol currency.Pair, l
 
 	params := url.Values{}
 	params.Set("symbol", symbolValue)
+	if limit == 0 {
+		// default to help select rate limits when no limit is provided
+		limit = 500
+	}
 	strLimit := strconv.FormatInt(limit, 10)
```

Comment:
This defaults to 500 we don't need to set this if limit is zero

Reply:
Do you mean do this instead?
```go
	if limit == 0 {
		// default
		limit = 500
	}
	strLimit := strconv.FormatInt(limit, 10)
	if !slices.Contains(uValidOBLimits, strLimit) {
		return nil, fmt.Errorf("invalid limit: %v", limit)
	}
	params.Set("limit", strLimit)
```

Reply:
Where does it do this?